.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.4"
```

Comment:
Check if macro version also allowed : 3.4.3, so that exact same version used as bosh release.

Reply:
Done